doc/library/config.txt

```diff
@@ -452,31 +456,38 @@ import theano and print the config variable, as in:
         automatically to get more memory. But this can cause
         fragmentation, see note above.
 
-.. attribute:: config.gpuarray.preallocate
+.. attribute:: config.lib.cnmem
 
-    Float value
+    .. note::
 
-    Default: 0
+        This value allocates GPU memory ONLY when using (:ref:`cuda`)
+        and has no effect when the GPU backend is (:ref:`gpuarray`).
+        For the new backend, please see ``config.gpuarray.preallocate``
 
-    Controls the preallocation of memory with the gpuarray backend.
+    Float value: >= 0
 
-    The value represents the start size (either in MB or the fraction
-    of total GPU memory) of the memory pool. If more memory is needed,
-    Theano will try to obtain more, but this can cause memory
-    fragmentation.
+    Controls the use of `CNMeM <https://github.com/NVIDIA/cnmem>`_ (a
+    faster CUDA memory allocator). Applies to the old GPU backend 
+    :ref:`cuda` up to Theano release 0.8.
 
-    A negative value will completely disable the allocation cache.
-    This can have a severe impact on performance and so should not be
-    done outside of debugging.
+    The CNMeM library is included in Theano and does not need to be
+    separately installed.
 
-        * < 0: disabled
-        * 0 <= N <= 1: use this fraction of the total GPU memory (clipped to .95 for driver memory).
+    The value represents the start size (either in MB or the fraction of total GPU
+    memory) of the memory pool. If more memory is needed, Theano will
+    try to obtain more, but this can cause memory fragmentation.
+
+        * 0: not enabled.
+        * 0 < N <= 1: use this fraction of the total GPU memory (clipped to .95 for driver memory).
         * > 1: use this number in megabytes (MB) of memory.
 
+
+    Default: 0 (but should change later)
```

Comment:
I think one of my comments got lost. Remove the comment in parenthesis. We won't change it.

Otherwise, it is good.